lxd/daemon.go

```diff
@@ -470,21 +472,38 @@ SELECT fingerprint FROM images WHERE cached=1 AND last_use_date<=strftime('%s',
 
 // StartDaemon starts the shared daemon with the provided configuration.
 func startDaemon() (*Daemon, error) {
-	d := &Daemon{}
+	d := &Daemon{
+		IsMock:  false,
+		Sockets: []net.Listener{},
+	}
+
+	if err := d.Init(); err != nil {
+		return nil, err
+	}
+
+	return d, nil
+}
 
+func (d *Daemon) Init() error {
 	/* Setup logging */
 	if shared.Log == nil {
 		shared.SetLogger("", "", true, true)
 	}
 
-	shared.Log.Info("LXD is starting")
+	if !d.IsMock {
+		shared.Log.Info("LXD is starting.",
+			log.Ctx{"path": shared.VarPath("")})
+	} else {
+		shared.Log.Info("Mock LXD is starting.",
+			log.Ctx{"path": shared.VarPath("")})
+	}
```

Comment:
Those strings shouldn't end with a final dot. That's a regression from the change I've pushed earlier today.